src/utils/WebResource.js

```diff
@@ -0,0 +1,9 @@
+window.tacticsUtils = window.tacticsUtils || {};
+
+(function () {
+    'use strict';
+
+    tacticsUtils.getWebResourceJSON = function (url) {
+        return fetch(url).then(response => response.json());
```

Comment:
I enjoy the simplicity, but it was actually kinda nice that you reject the promise when the response is not "ok".  One of the nice features about fetch is that it does not automatically raise an exception just because the status indicates a failure.  Sometimes a failure is just data.  But in this case, a failure is an exception.  So, could test the value of "response.ok" and return the JSON or throw an error depending on its value?